HelpSource/Classes/ServerOptions.schelp

```diff
@@ -142,6 +142,11 @@ A Boolean indicating whether the server should try to lock its memory into physi
 method:: maxLogins
 An Integer indicating the maximum number of clients which can simultaneously receive notifications from the server. When using TCP this is also the maximum number of simultaneous connections. This is also used by the language to split ranges of link::Classes/Node##Nodes::, link::Classes/Buffer##Buffers::, or link::Classes/Bus##Busses::. In multi-client situations you will need to set this to at least the number of clients you wish to allow. This must be the same in the Server instances on every client. The default is 1.
 
+method:: safetyClipThreshold
+note:: MacOS only::
+A Float indicating a safety threshold for output values to be clipped to. This is necessary on macOS because setting a low system volume doesn't prevent output values greater than +/- 1 (which can happen by mistake, e.g. when sending a negative coefficient to a filter) to sound extremely loud, since CoreAudio applies only a relative attenuation. With this threshold, values are clipped just before being written to hardware output busses, so although this affects routing to other apps, it doesn't affect recording, which still benefits from the full 32-bit float range without any clipping.
+
+Defaults to a threshold of 1.26 (ca. 2 dB), to save some ears and still allow some headroom. Setting safetyClipThreshold to strong::inf:: disables clipping altogether.
```

Comment:
I tested this and it works as desired, thank you so much @elgiano and @brianlheim ! This is a HUGE improvement!

Before merging I wanted to propose a small change to the documentation:

> This is necessary on macOS because setting a low system volume doesn't prevent output values greater than +/- 1 (which can happen by mistake, e.g. when sending a negative coefficient to a filter) to sound extremely loud, since CoreAudio applies only a relative attenuation. 

to sound extremely loud -> from sounding extremely loud

and maybe changing the order and dropping the part about CoreAudio:

>This is necessary on macOS because setting a low system volume doesn't prevent output values greater than +/- 1 from sounding extremely loud, which can happen by mistake, e.g. when sending a negative coefficient to a filter. 

Regarding this:

> With this threshold, values are clipped just before being written to hardware output busses, so although this affects routing to other apps, it doesn't affect recording, which still benefits from the full 32-bit float range without any clipping.

Routing sound between apps on macOS is more of an exception than a rule. And also server recording might or might not be set to 32-bit float. How about rephrasing this as:

> With this threshold, values are clipped just before being written to hardware output busses, which does not affect the recording. However, the signal will be affected if it's above the threshold and  the sound is routed to other apps using 3rd-party software.

And finally:

> Defaults to a threshold of 1.26 (ca. 2 dB), to save some ears and still allow some headroom. Setting safetyClipThreshold to strong::inf:: disables clipping altogether.

I think it's both `inf` and `0`, right? I'd list both here. I'd also use `code:: ::` instead of `strong:: ::` for it.

Reply:
Thanks @dyfer, your changes make it look clearer to me, I just included them all